src/testData.ts

```diff
@@ -0,0 +1,49 @@
+
+import * as Types from './types'
+
+export const sectionData: Types.sectionData = {
+  name: 'Intro',
+  description: 'This intro section consists of a tuba quartet in the style of DJ Templeton & The Windsurfers',
+  numRevisions: 42,
+};
+
+export const currentChordProgression: Types.chordProgression = ['C', 'Dm', 'F', 'G']
+
+export const files: Types.file[] = [
+  {
+    title: 'Bass.mp3',
+    numComments: 2,
+    id: 'change me to something better 0 '
```

Comment:
I think just regular `0` is fine here for now. It's somewhat realistic I think. You can use this to create a random string in the meantime:

```suggestion
    id: Math.random().toString().slice(2),
```